include/chemistry/bcl_chemistry_fragment_add_med_chem.h

```diff
@@ -68,9 +68,16 @@ namespace bcl
       util::ShPtr< FragmentEnsemble> m_FragmentPool;
       std::string m_MedChemFilename;
 
+      //! alternate element types to Undefined to control directionality
+      std::string m_MedChemFragmentLinkElementType;
```

Comment:
Why aren't these chemistry::ElementType ? or lists of them if that's more appropriate?
Not that performance is an issue here, but it does lots of needless lookups later where it would seem better to just have an element type instance unless I'm missing something